file-mon/src/main/java/com/wipro/ats/bdre/filemon/FileMonitor.java

```diff
@@ -63,8 +64,9 @@ public static FileMonitor getInstance() {
 
     //Read the monitored directories, file patterns, subprocessIds and serverIds and build the HashTable
     private void init() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "dirList" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)